Makefile

```diff
@@ -187,6 +180,29 @@ oracle-database:
 		-v $(PWD)/script/oracle:/opt/oracle/scripts/setup \
 		quay.io/3scale/oracle:12.2.0.1-ee
 
+oracle-db-setup: ## Creates databases in Oracle
+oracle-db-setup: oracle-database
+	MASTER_PASSWORD=p USER_PASSWORD=p ORACLE_SYSTEM_PASSWORD=threescalepass NLS_LANG='AMERICAN_AMERICA.UTF8' DATABASE_URL="oracle-enhanced://rails:railspass@127.0.0.1:1521/systempdb" bundle exec rake db:drop db:create db:setup
+
+## Postgres database containerization
+postgres-database: ## Starts Postgres database container
+postgres-database:
+	[ "$(shell docker inspect -f '{{.State.Running}}' postgres-database 2>/dev/null)" = "true" ] || docker start postgres-database || docker run \
+		-p 5432:5432 \
+		--name postgres-database \
+		-e POSTGRES_USER=postgres \
+		-e POSTGRES_DB=threescale \
+		circleci/postgres:10.5-alpine
+
+postgres-db-setup: ## Creates databases in Postgres
+postgres-db-setup: postgres-database
+	DATABASE_URL="postgresql://postgres:@127.0.0.1:5433/threescale" bundle exec rake db:drop db:create db:setup
+
+schema: ## Runs db schema migrations. Run this when you have changes to your database schema that you have added as new migrations.
+	bundle exec rake db:migrate db:schema:dump
+	MASTER_PASSWORD=p USER_PASSWORD=p ORACLE_SYSTEM_PASSWORD=threescalepass NLS_LANG='AMERICAN_AMERICA.UTF8' DATABASE_URL="oracle-enhanced://rails:railspass@127.0.0.1:1521/systempdb" bundle exec rake db:migrate db:schema:dump
```

Comment:
Does this work? In my case it requires me to send all these env vars: `DATABASE_URL="oracle-enhanced://rails:railspass@127.0.0.1:1521/systempdb" ORACLE_SYSTEM_PASSWORD=threescalepass NLS_LANG=AMERICAN_AMERICA.UTF8`